Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityGazeProvider.cs

```diff
@@ -15,6 +15,16 @@ public interface IMixedRealityGazeProvider
         /// </summary>
         bool Enabled { get; set; }
 
+        /// <summary>
+        /// The <see cref="IMixedRealtyInputSystem"/> instance for which this object is providing gaze data.
+        /// </summary>
+        IMixedRealityInputSystem InputSystem { set; }
+
+        /// <summary>
+        /// The transform of the Mixed Reality Playspace
+        /// </summary>
+        Transform Playspace { set; }
+
```

Comment:
why no get?

Reply:
i didnt see a need for one. i am happy to add it for balance.